<ui version="4.0" >
 <class>F_Yanlislar</class>
 <widget class="QWidget" name="F_Yanlislar" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Yapılan Yanlışlar</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>../../../../../../usr/kde/3.5/share/icons/crystalsvg/32x32/actions/messagebox_critical.png</normaloff>../../../../../../usr/kde/3.5/share/icons/crystalsvg/32x32/actions/messagebox_critical.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QPushButton" name="PbHepsi" >
     <property name="text" >
      <string>Tüm Cevapladığım Soruları Göster</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QTextEdit" name="textEdit" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QToolButton" name="tbYazdir" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor" >
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="icon" >
      <iconset resource="ikonlar.qrc" >
       <normaloff>:/ikonlar/ikonlar/agt_print.png</normaloff>:/ikonlar/ikonlar/agt_print.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="popupMode" >
      <enum>QToolButton::DelayedPopup</enum>
     </property>
     <property name="toolButtonStyle" >
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ikonlar.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>tbYazdir</sender>
   <signal>clicked()</signal>
   <receiver>F_Yanlislar</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>34</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
